.github/workflows/release.yml

```diff
@@ -63,3 +63,39 @@ jobs:
         with:
           user: __token__
           password: ${{ secrets.PYPI_TOKEN_DEEPRANK2 }}
+
+  read_version:
+    needs: upload_pypi
+    name: Read version from TOML
```

Comment:
Maybe rename this to "read-only version"? or did I misunderstand?

Reply:
Here the version number is read in order to set the same number for the docker image. It doesn't make much difference imo, but I've changed it to `read_only_ version`.